npm doesn't work after updating winpty-git

winpty exact version is winpty-git-1.1.1.184.db12b07

Steps to reproduce
------------------------
1. Download msys2-base-x86_64-20150916.tar.xz
2. Unpack it to d:/soft/msys2
3. Run mingw64-shell.bat
4. Restart
5. pacman -S mingw64/mingw-w64-x86_64-nodejs
6. npm (output should be ok)
7. update-core
8. Restart
9. pacman -S msys/winpty-git
10. npm

Expected result
-------------------
npm help output as usual

Actual result
----------------
````
module.js:340
    throw err;
          ^
Error: Cannot find module 'D:\mingw64\lib\node_modules\npm\bin\npm-cli.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
````
Additional info
-----------------
npm looks for npm-cli.js in   
````
"$basedir/node" "$basedir/../lib/node_modules/npm/bin/npm-cli.js" "$@"
````
Basedir is /mingw64/bin, so the problem must be evaluating relative path

Downgrading to winpty-git-1.1.1.148.47a69d0 helps

--- a/winpty-git/0002-Add-support-for-conversion-of-all-argv-path-args.patch
+++ b/winpty-git/0002-Add-support-for-conversion-of-all-argv-path-args.patch
@@ -1,0 +1,25 @@
+From 2f9c59cf632f0b01b81757d354162d2570f1d5c4 Mon Sep 17 00:00:00 2001
+From: David Macek <david.macek.0@gmail.com>
+Date: Thu, 8 Oct 2015 21:38:59 +0200
+Subject: [PATCH] Add support for conversion of all argv path args
+
+---
+ unix-adapter/main.cc | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/unix-adapter/main.cc b/unix-adapter/main.cc
+index 7e75d28..6c34fd9 100644
+--- a/unix-adapter/main.cc
++++ b/unix-adapter/main.cc
+@@ -377,7 +377,7 @@ int main(int argc, char *argv[])
+         std::vector<std::string> argVector;
+         argVector.push_back(convertPosixPathToWin(argv[1]));
+         for (int i = 2; i < argc; ++i)
+-            argVector.push_back(argv[i]);
++            argVector.push_back(convertPosixPathToWin(argv[i]));
+         std::string cmdLine = argvToCommandLine(argVector);
+         wchar_t *cmdLineW = heapMbsToWcs(cmdLine.c_str());
+         int ret = winpty_start_process(winpty,
+-- 
+2.5.2
+

--- a/winpty-git/PKGBUILD
+++ b/winpty-git/PKGBUILD
@@ -4,7 +4,7 @@ _realname=winpty
 pkgname="${_realname}-git"
 _ver_base=1.1.1
 pkgver=1.1.1.184.db12b07
-pkgrel=1
+pkgrel=2
 pkgdesc="A Windows software package providing an interface similar to a Unix pty-master for communicating with Windows console programs"
 arch=('i686' 'x86_64')
 url="https://github.com/rprichard/winpty"
@@ -17,9 +17,11 @@ conflicts=("${_realname}")
 options=('staticlibs' 'strip')
 
 source=("${_realname}"::"git+https://github.com/rprichard/winpty.git"
+        "0002-Add-support-for-conversion-of-all-argv-path-args.patch"
         "0003-rename-console.exe-to-winpty.exe-for-mintty-unicode-.patch")
 
 md5sums=('SKIP'
+         'e352e89e98422fcdbc0182b99a22dccf'
          'a86ac650f9893ed1c6a606b803e684aa')
 
 
@@ -31,6 +33,7 @@ pkgver() {
 prepare() {
   cd "${srcdir}/${_realname}"
 
+  git am "$srcdir/0002-Add-support-for-conversion-of-all-argv-path-args.patch"
   git am "$srcdir/0003-rename-console.exe-to-winpty.exe-for-mintty-unicode-.patch"
 }
 

